mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -316,15 +320,15 @@ class PopulateParamsBase {
                                                   bool isGemmA,
                                                   DerivedParams &derived) {
 
-    bool gemmPos1Vectorizable = false;
+    bool gemmKVectorizable = false;
     int64_t vectorizableLength = 0;
     if (isGemmA) {
       obtainGemmADimKVectorizable(ctx.opType, ctx.dimIndexVal,
```

Comment:
I didn't review what's happening inside ```obtainGemmADimKVectorizable```, etc

Reply:
it's okay for now. but we will need to review them in subsequent PRs.